lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -44,39 +57,49 @@ function getAllLoadedFonts(descriptors) {
 
 /**
  * Collect authored webfont data from the `CSSFontFaceRule`s present in document.styleSheets
- * @return {{}}
+ * @return {Promise<Array<LH.Artifacts.Font|FontGatherError>>}
  */
 /* istanbul ignore next */
 function getFontFaceFromStylesheets() {
   /**
    * Get full data about each CSSFontFaceRule within a styleSheet object
-   * @param {StyleSheet} stylesheet
-   * @return {{}}
+   * @param {CSSStyleSheet} stylesheet
+   * @return {Array<LH.Artifacts.Font>}
    */
   function getSheetsFontFaces(stylesheet) {
     const fontUrlRegex = 'url\\((?:")([^"]+)(?:"|\')\\)';
     const fontFaceRules = [];
+
     if (stylesheet.cssRules) {
-      for (const rule of stylesheet.cssRules) {
+      for (const rule of Array.from(stylesheet.cssRules)) {
         if (rule instanceof CSSFontFaceRule) {
           const fontsObject = {
+            // @ts-ignore (currently) non-standard Chrome extension to CSSStyleDeclaration
             display: rule.style.fontDisplay || 'auto',
-            family: rule.style.fontFamily.replace(/"|'/g, ''),
+            family: rule.style.fontFamily ? rule.style.fontFamily.replace(/"|'/g, '') : '',
             stretch: rule.style.fontStretch || 'normal',
             style: rule.style.fontStyle || 'normal',
             weight: rule.style.fontWeight || 'normal',
             variant: rule.style.fontVariant || 'normal',
+            // @ts-ignore (currently) non-standard Chrome extension to CSSStyleDeclaration
```

Comment:
i dont think these are non-standard, they're just fairly new.

Reply:
adding link to that discussion